blocks/post/edit.tsx

```diff
@@ -92,6 +95,47 @@ export default function Edit({
     });
   };
 
+  const isShown = (item: string) => {
+    switch (item) {
+      case 'image':
+        return showImage;
+      case 'title':
+        return true;
+      case 'byline':
+        return showByline;
+      case 'excerpt':
+        return showExcerpt;
+      case 'content':
+        return showContent;
+      case 'cta':
+        return showCta;
+      default:
+        return false;
+    }
+  };
+
+  const toggleShown = (item: string) => {
+    switch (item) {
+      case 'image':
+        setAttributes({ showImage: !showImage });
+        break;
+      case 'byline':
+        setAttributes({ showByline: !showByline });
+        break;
+      case 'excerpt':
+        setAttributes({ showExcerpt: !showExcerpt });
+        break;
+      case 'content':
+        setAttributes({ showContent: !showContent });
+        break;
+      case 'cta':
+        setAttributes({ showCta: !showCta });
+        break;
+      default:
+        break;
+    }
+  };
+
```

Comment:
Also for the ternary's in this file we may consider logical nullish
`let postTitle = record?.title.rendered ?? '';`
in place of 
`let postTitle = record ? record.title.rendered : '';`